addons/point_of_sale/static/tests/tours/ReceiptScreen.tour.js

```diff
@@ -18,11 +18,16 @@ registry.category("web_tour.tours").add("ReceiptScreenTour", {
             // press close button in receipt screen
             ProductScreen.addOrderline("Letter Tray", "10", "5"),
             ProductScreen.selectedOrderlineHas("Letter Tray", "10"),
+            ProductScreen.clickPartnerButton(),
+            ProductScreen.clickCustomer("Addison Olson"),
             ProductScreen.clickPayButton(),
             PaymentScreen.clickPaymentMethod("Bank"),
             PaymentScreen.validateButtonIsHighlighted(true),
+            PaymentScreen.clickShipLaterButton(),
```

Comment:
Before validating the order, could you insert a step that checks if the ship later button is highlighted? I'm not completely sure of the proposed trigger, I'll leave it to you.
```suggestion
            PaymentScreen.clickShipLaterButton(),
            {
                trigger: ".payment-buttons button:contains(Ship Later).highlight"
                isCheck: true,
            },
```